code/modules/detectivework/tools/sample_kits/_sample.dm

```diff
@@ -0,0 +1,56 @@
+
+/obj/item/forensics/sample
+	name = "forensic sample"
+	icon = 'icons/obj/forensics.dmi'
+	var/object
+	var/list/possible_evidence_types = list(/datum/forensics/fibers)
+	var/list/evidence
+
+/obj/item/forensics/sample/Initialize(mapload, var/atom/supplied)
+	. = ..(mapload)
+	if(supplied)
+		copy_evidence(supplied)
+		name = "[initial(name)] (\the [supplied])"
+		object = "[supplied], at [get_area(supplied)]"
+	update_icon()
+
+/obj/item/forensics/sample/examine(mob/user, distance)
+	. = ..()
+	if(distance <= 1 && object)
+		to_chat(user, "The label says: '[object]'")
+
+/obj/item/forensics/sample/proc/copy_evidence(var/atom/supplied)
+	var/datum/extension/forensic_evidence/forensics = get_extension(supplied, /datum/extension/forensic_evidence)
+	if(forensics)
+		for(var/T in possible_evidence_types)
+			var/datum/forensics/F = forensics.evidence[T]
+			if(F)
+				LAZYADD(evidence, F.copy())
+				if(F.remove_on_transfer)
+					forensics.remove_data(T)
+
+/obj/item/forensics/sample/proc/merge_evidence(var/obj/item/forensics/sample/supplied, var/mob/user)
+	if(!LAZYLEN(supplied.evidence))
+		return 0
+	// no merging prints with fibers
+	if(!same_entries(possible_evidence_types, supplied.possible_evidence_types))
+		return
+	merge_evidence_list(supplied.evidence)
+	SetName("[initial(name)] (combined)")
+	object = supplied.object + ", " + object
+	to_chat(user, SPAN_NOTICE("You combine \the [supplied] into \the [src]."))
+	return 1
+	
+/obj/item/forensics/sample/proc/merge_evidence_list(var/list/new_evidence)
+	LAZYDISTINCTADD(evidence, new_evidence)
+
+/obj/item/forensics/sample/resolve_attackby(atom/A, mob/user, var/click_params)
+	// Fingerprints will be handled in after_attack() to not mess up the samples taken
```

Comment:
Noting that this is questionable at best, as afterattack may or may not be called. Suggest adding prints after attackby if the return value is true (afterattack will not be called then).

Reply:
axed it, it's done by an item flag nowadays and these objects have it set.